internal/sink/sink.go

```diff
@@ -27,84 +30,141 @@ import (
 )
 
 type Sink interface {
-	Update(ancla.Register) error
-	Send(*ring.Ring, string, string, *wrp.Message) error
+	Send(string, string, *wrp.Message) error
 }
 
 type WebhookV1 struct {
-	id               string
-	deliveryInterval time.Duration
-	deliveryRetries  int
-	logger           *zap.Logger
+	urls *ring.Ring
+	CommonWebhook
 	//TODO: need to determine best way to add client and client middleware to WebhooV1
 	// client           http.Client
 	// clientMiddleware func(http.Client) http.Client
 }
 
+type Webhooks []*WebhookV1
```

Comment:
going to use this webhookV1 struct (will be renaming this so as not to be confused with the webhookV1 schema)  as the struct for the newest webhook struct in webhook schema for now - might have to add a third struct to satisfy all registrations